src/soorgeon/cli.py

```diff
@@ -1,7 +1,9 @@
 import click
 import tempfile
 import jupytext
-from os.path import abspath, dirname
+import papermill as pm
```

Comment:
@94rain please consume our ploomber engine instead of Papermill (it has more capabilities on top).

Reply:
our engine still needs some testing, let's add this to the next cycle

Reply:
I think we can open a story once it's stable and change in all repos. Also I don't want to block this PR since it'll take some time to get to a stable version.

Reply:
yeah, we won't block this PR, we'll merge it with papermill